notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb

```diff
@@ -0,0 +1,702 @@
+{
```

Comment:
<p>ttps://drive.google.com/drive/folders/1BO_dyz-p65qhBRRMRA4TbZ8qW4rB99JZ -&gt; use bucket GCS bucket</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
This google drive is a public drive created by OSS community, we don't want to host them in our GCS bucket.
See https://huggingface.co/docs/diffusers/training/dreambooth#dog-toy-example